src/ai_marketplace_monitor/cli.py

```
@@ -114,6 +140,36 @@ def main(
+                removed = systemd_service.uninstall_service()
+                if removed is None:
+                    logger.info(
+                        f"""{hilight("[Service]", "info")} No systemd user unit was installed; nothing to remove."""
+                    )
+                else:
+                    logger.info(
+                        f"""{hilight("[Service]", "succ")} Removed systemd user unit {hilight(str(removed), "name")}."""
+                    )
+            if service_status:
+                rich.print(systemd_service.service_status())
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)